doc/quantum_error_correction.rst

```diff
@@ -1,3 +1,7 @@
+.. default-role:: math
+
+.. _quantum-error-correction:
```

Comment:
This should probably be added to the toc, so that it's reachable in the docs.

Reply:
done!